### ISSUE TITLE
Ctrl Click on FROM leads to a DockerHub link, instead of a stage in a multi-stage config

### ISSUE THREAD

**Issue by samhwang** [microsoft/vscode-docker#2043]
Hey VSCode Docker team,

I just noticed that the recent update has made it possible to Ctrl+Click on the image name, and then open an appropriate one in DockerHub, which is *fantastic!* . So I just encountered this little issue, and absolutely not a deal breaker at all, but it would be nice to have it fixed.

## The situation:
I want all my stages to have the same `WORKDIR` and `LABEL`, and `EXPOSE`-ing the same port, so I put an official image at the top, and made it a `base` stage. A snippet of what I'm working is like this.

```
FROM node:12-alpine as base
LABEL MAINTAINER="Sam Huynh"
WORKDIR /srv/app
EXPOSE 1337

# Dev dependencies
FROM base as dev_dependencies
COPY package.json yarn.lock ./
RUN yarn install && \
    yarn cache clean

# Dev stage
FROM base as development

COPY . .
COPY --from=dev_dependencies /srv/app/node_modules ./node_modules
CMD ["yarn", "develop"]

# Other stages
## Build something else...
```

## Working:
- Ctrl+Clicking on `node` in `FROM node:12-alpine` prompts me to open [The official Node docker image](https://hub.docker.com/_/node), which is expected.
![Screenshot from 2020-06-05 10-24-11](https://user-images.githubusercontent.com/4188758/83823212-b79f2200-a716-11ea-947c-db338f67f4d5.png)

- Ctrl+clicking on `dev_dependencies` in `--from=dev_dependencies` brings me back to the that stage, which is also correct.
![Screenshot from 2020-06-05 10-27-08](https://user-images.githubusercontent.com/4188758/83823339-22e8f400-a717-11ea-888d-1f4e2215ee48.png)

## Expected:
- Ctrl+Clicking at `base` in `FROM base` goes back to the `base` stage.

## What happened:
- Ctrl+Clicking at `base` in `FROM base` prompts me to open [The "offical" base docker image](https://hub.docker.com/_/base), which doesn't exist.
![Screenshot from 2020-06-05 10-30-10](https://user-images.githubusercontent.com/4188758/83823526-92f77a00-a717-11ea-9056-17e53a835bb6.png)

Please let me know if there are any other info I can put in here.

**Comment by rcjsuen** [comment]
> I just noticed that the recent update has made it possible to Ctrl+Click on the image name, and then open an appropriate one in DockerHub, which is _fantastic!_ .

Interesting, that feature should have been there for at least a year now... 🤔 Well, glad you find it useful, @samhwang!

As to your issue, thank you for bringing this to my attention. I have opened two tickets to address this.
- https://github.com/rcjsuen/dockerfile-language-service/issues/67
- https://github.com/rcjsuen/dockerfile-language-service/issues/68

**Comment by samhwang** [comment]
Yeah I just got it working at least a month ago while I started working on some Dockerfiles for another project, and I swear I never saw it before March. :P

**Comment by rcjsuen** [comment]
@samhwang I have opened #2146 to address this. Thanks again for reporting this bug!

**Comment by bwateratmsft** [comment]
We have released Docker 1.4.0 which contains a fix for this.
